config/initializers/devise.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 Devise.setup do |config|
-  config.mailer_sender = ENV.fetch("SMTP_USERNAME", "participagencat@gencat.cat")
```

Comment:
do we have this default in the secrets.yml?

Reply:
I have changed it to the decidim sender